src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseAttributes.java

```diff
@@ -51,13 +51,13 @@ public FeedbackResponseAttributes() {
 
     public FeedbackResponseAttributes(String feedbackSessionName,
             String courseId, String feedbackQuestionId,
-            FeedbackQuestionType feedbackQuestionType, String giverEmail, String giverSection,
+            FeedbackQuestionType feedbackQuestionType, String giver, String giverSection,
             String recipientEmail, String recipientSection, Text responseMetaData) {
```

Comment:
@damithc  Yes,  we should as recipientEmail was change to recipient in https://github.com/TEAMMATES/teammates/pull/5547. Some might have left.